README.md

```diff
@@ -13,16 +13,27 @@ issues, mailing lists, conferences, etc.
 
 For more specific topics, try a SIG.
 
-## SIGs
+## Governance
 
-Kubernetes is a set of subprojects, each shepherded by a Special Interest Group (SIG).
+Kubernetes has three types of groups that are officially supported:
 
-A first step to contributing is to pick from the [list of kubernetes SIGs](sig-list.md).
+* **Committees** are named sets of people that are chartered to take on sensitive topics.
+  This group is encouraged to be as open as possible while achieving its mission but, because of the nature of the topics discussed, private communications are allowed.
+  Examples of committees include the steering committee and things like security or code of conduct.
+* **Special Interest Groups (SIGs)** are persistent open groups that focus on a part of the project.
+  SIGs must have open and transparent proceedings.
+  Anyone is welcome to participate and contribute provided they follow the Kubernetes Code of Conduct.
+  The purpose of a SIG is to own and develop a set of **subprojects**.
+  * **Subprojects** Each SIG can have a set of subprojects.
+    These are smaller groups that can work independently.
+    Some subprojects will be part of the main Kubernetes deliverables while others will be more speculative and live in the `kubernetes-sigs` github org.
+* **Working Groups** are temporary groups that are formed to address issues that cross SIG boundaries.
+  Working groups do not own any code or other long term artifacts.
```

Comment:
See @philips comment below.  Clarify to make it clear that prototypes and proof-of-concept implementations by working groups are in scope and encouraged.

Reply:
We have a repository policy:
https://github.com/kubernetes/community/blob/master/github-management/kubernetes-repositories.md

This should not change it.